cluster/pulumi/infra/src/auth0.ts

```diff
@@ -36,6 +37,7 @@ function ledgerApiAudience(
         name: `Ledger API for SV ${svNamespace} on ${clusterBasename} (Pulumi managed)`,
         identifier: `https://ledger_api.${svNamespace}.${clusterBasename}.canton.network`,
         allowOfflineAccess: true, // TODO(DACH-NY/canton-network-internal#2114): is this still needed?
+        tokenLifetime: 2592000, // 30 days // TODO(DACH-NY/canton-network-internal#2114): make this configurable? We want it to be long for fixed token clusters
```

Comment:
Shouldn't this only apply for the clusters that use fixed tokens? It's not quite clear to me that this is the case